index.html

```diff
@@ -10,16 +10,25 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
-    <header class="app-header">
-        <nav>
-            <a href="#">My Logo</a>
-            <i class="fa fa-bars fa-2x bars "></i>
-            
-                <ul class="menu">
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
-                </ul>
```

Comment:
- [OPTIONAL] Kindly make sure that you also have those options in your desktop version it will make your pages easy to navigate 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/224431290-9799cb20-450d-450a-9832-8129203ea220.png)


## Figma design 

![image](https://user-images.githubusercontent.com/95297251/224431167-d69276c8-bdaa-4dd0-9fa2-20ac9f084d7e.png)